src/constants.tsx

```diff
@@ -0,0 +1,43 @@
+//Constants for initial useState, including typical 52 card deck
+const deck52 = [
```

Comment:
This should be generated in a nested for loop.

we should also make an enumeration for the suits and another for the ranks so we can have more readable names when writing code or testing.